app/views/catalog/_index_brief.html.erb

```diff
@@ -1,51 +1,50 @@
-<%
-  preview_container = 'preview-container-' + document.id
-  preview_attrs = preview_data_attrs("preview-brief", document[:id], '.' + preview_container)
-%>
 <% doc_presenter = document_presenter(document) %>
-<div class=" brief-document container-fluid" <%= preview_attrs %>>
-  <div class="row row-eq-height brief-container">
-    <div class="col-md-10">
-      <h3 class="index_title">
-        <% counter = document_counter_with_offset(document_counter) %>
-        <%= render_resource_icon doc_presenter.formats %>
-        <%= document.online_label %>
-        <span class="document-counter">
-          <%= t('blacklight.search.documents.counter', :counter => counter) if counter %>
-        </span>
-        <%= link_to_document document, counter: (counter + @response.params[:start].to_i) %>
-        <span class="main-title-date"><%= get_main_title_date(document) %></span>
-      </h3>
-      <% if document[:vern_title_display].present? %>
-        <ul class="document-metadata"><li><%= document[:vern_title_display].html_safe %></li></ul>
-      <% end %>
-      <% if document.holdings.libraries.select(&:present?).any? %>
-        <ul class='list-unstyled'>
-          <% document.holdings.libraries.select(&:present?).each do |library| %>
-            <% library.locations.uniq(&:name).each do |location| %>
-                <% callnumber = "(no call number)" %>
-              <% if location.items.present? %>
-                <% callnumber = "#{location.items.first.callnumber}" %>
+<div data-controller="preview-brief" data-preview-brief-url-value="<%= preview_path(document[:id]) %>">
```

Comment:
A lot of this is just an indent change. It's easier to review if you "Hide whitespace"